lib/collection.js

```diff
@@ -324,7 +338,7 @@ Collection = Events.inherit(/** @lends Collection.prototype*/{
         } else {
             return new this.modelType(model, {
                 collection: this,
-                isNested: true
+                owner: this._owner
```

Comment:
I think collection should be owner in this case instead of parent model of the collection